boards/qn9080dk/include/periph_conf.h

```diff
@@ -43,6 +43,28 @@ static const i2c_conf_t i2c_config[] = {
 #define I2C_NUMOF           ARRAY_SIZE(i2c_config)
 /** @} */
 
+/**
+ * @name   SPI configuration
+ * @{
+ */
+static const spi_conf_t spi_config[] = {
+    {
+        .dev            = SPI0,  /* Flexcomm 2 */
+        .copi_pin       = GPIO_PIN(PORT_A, 4),
+        .cipo_pin       = GPIO_PIN(PORT_A, 5),
```

Comment:
```suggestion
        .cipo_pin       = GPIO_PIN(PORT_A, 5),
        .copi_pin       = GPIO_PIN(PORT_A, 4),
```

In C++, fields must be initialized in the same order as they are declared.
This should fix the build failure with `tests/cpp11_mutex`

Reply:
How about adding `-Wno-missing-field-initializers` to `CXXFLAGS` instead? IMO the whole point in using member names during the initialization is to future proof your code, so that reordering members or adding members to a `struct` won't fall on your foot.

Reply:
But that would be something for another PR - I'd just go with the straightforward solution here.